.github/workflows/cache-issues.yaml

```diff
@@ -0,0 +1,33 @@
+name: Cache issues status
```

Comment:
is there some retry mechanism when it fails? 
Will we be informed when it fails and how?

Reply:
no retry, and it's visible on the actions page,

I think admin would get notification, but not sure.

Reply:
we'll see, maybe as a follow-up we'll add a step to send mail notification,
but it's a bit more work to configure such a thing. (need to pass gmail cred in)

Reply:
ok, let's see how it looks in practice and adapt as needed